### PR TITLE
Initial take on web interface

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

File: satnogsclient/web/static/css/app.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+body {
+  padding-top: 50px;
+}
+.settings {
+  padding: 40px 15px;
+}
```
Comment (comzeradd) [inline satnogsclient/web/static/css/app.css:6]: If we want to have unified code style across subprojects, we could go with 4 spaces in css. See [Network's editorconfig](https://github.com/satnogs/satnogs-network/blob/dev/.editorconfig).

File: satnogsclient/web/templates/index.j2 [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>SatNOGS client</title>
```
Comment (comzeradd) [inline satnogsclient/web/templates/index.j2]: I would capitalize `client` first letter.

File: satnogsclient/web/templates/index.j2 [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    <!-- CSS -->
+    <link href="{{ url_for('static', filename='css/bootstrap.min.css') }}" rel="stylesheet">
+    <link href="{{ url_for('static', filename='css/app.css') }}" rel="stylesheet">
+  </head>
+
+  <body>
+    <nav class="navbar navbar-inverse navbar-fixed-top">
+      <div class="container">
+        <div class="navbar-header">
+          <a class="navbar-brand" href="#">SatNOGS client</a>
```
Comment (comzeradd) [inline satnogsclient/web/templates/index.j2]: Same here.

**Comment by johngian** [comment]
@comzeradd PR updated

**Comment by comzeradd** [comment]
Looks good. Go for it! :beers:

**Pull request merged**